fiftyone/utils/clip/zoo.py

```diff
@@ -187,7 +187,7 @@ def _predict_all(self, imgs):
         frame_size = (width, height)
 
         if self._using_gpu:
-            imgs = imgs.cuda()
+            imgs = imgs.to(torch.device(self.device))
```

Comment:
Looks like all of the models touched in this PR are subclasses of `TorchImageModel`, for which `self.device` is already a `torch.device()`:

https://github.com/voxel51/fiftyone/blob/84f7e27986a61b9c4b8c9e3dfc1ed032e55df42d/fiftyone/utils/torch.py#L530

So just `.to(self.device)` should suffice I believe?